frontend/src/main-components/InsideFolderContent.js

```diff
@@ -1,22 +1,66 @@
 import React from "react";
 import styled from "@emotion/styled";
+import Flashcard from "../flashcards/Flashcard.js";
+import NewCard from "../flashcards/NewCard.js";
 
+const DebugMessage = (message, isOn) => {
```

Comment:
Move this into a shared file?

Is the isOn parameter necessary? It might be better to just hardcode that value. The motivation behind this logging function is that when developing code, you can turn all logging on, but in production you can turn all logging off. Giving each invocation of `DebugMessage` the ability to override that is probably counterproductive.

Also, I would find a different name for `DebugMessage` that has "log" in it. Right now, the name `DebugMessage` seems like it's an object/constructor.